frontend/src/metabase/components/FieldValuesWidget/FieldValuesWidget.tsx

```diff
@@ -485,15 +485,22 @@ export function FieldValuesWidgetInner({
     valuesMode,
   });
 
+  const [isInitiliazing, setIsInitiliazing] = useState(isListMode);
```

Comment:
What issue does this solve? I can't figure it out.

Reply:
The multiselect component sometimes changes width when it has values in it and is focused. There's no way to turn this off.

This can make it tricky to click the `Add Filter` button (which is basically what was being complained about in the [initial issue that kicked of this series of PRs](https://github.com/metabase/metabase/issues/49279)).

This fix is a bit hacky (limit the width to the initial width on render). We could also just set a fixed width, but that might make the component less versatile.

Reply:
I still can't figure out how `isInitializing` helps with that. It definitely deserves a comment.

Reply:
Oops scratch that. I misread the line that this was referring to.

The component does not know ahead of time how it will render: for example, it might render as a search box (event when set to be a dropdown) if there are more than 1000 values. To determine how to render we need to fetch data first. 
We want to avoid switching between different versions of the component when the data loads, so we show a loading spinner before the initial load. For subsequent loads we can rely on the individual filter components to render their own spinner. 

`isInitializing` denotes the state of: we are loading for the first time.

Reply:
> The component does not know ahead of time how it will render: for example, it might render as a search box (event when set to be a dropdown) if there are more than 1000 values. To determine how to render we need to fetch data first. We want to avoid switching between different versions of the component when the data loads, so we show a loading spinner before the initial load. For subsequent loads we can rely on the individual filter components to render their own spinner.

Got it, thanks! This explanation would make a really useful comment in code.